ACE/ace/Logging_Strategy.cpp

```diff
@@ -442,7 +442,8 @@ ACE_Logging_Strategy::handle_timeout (const ACE_Time_Value &,
       // Close the current ostream.
 #if defined (ACE_LACKS_IOSTREAM_TOTALLY)
       FILE *output_file = (FILE *) this->log_msg_->msg_ostream ();
-      ACE_OS::fclose (output_file);
+      if (output_file)
+        ACE_OS::fclose (output_file);
```

Comment:
setting to nullptr only when it is non null?

Reply:
After this line, `output_file` is assigned before using.

Reply:
I meant the part `this->log_msg_->msg_ostream (nullptr);`

Reply:
OK